c++/src/kj/async-io.h

```diff
@@ -173,6 +173,20 @@ class AsyncIoStream: public AsyncInputStream, public AsyncOutputStream {
   virtual Maybe<void*> getWin32Handle() const { return nullptr; }
   // Get the underlying Win32 HANDLE, if any. Returns nullptr if this object actually isn't
   // wrapping a handle.
+
+#if _WIN32
```

Comment:
I don't think we need this `#if`. Both methods can be defined on all platforms, but the one that isn't relevant to the platform will always return null, same as the `get` methods above.

Reply:
As far as I can tell, `AutoCloseHandle` is defined only if _WIN32, that's why I've added if guard